Shared/Services/TranslationFavoritesService.swift

```diff
@@ -7,17 +7,21 @@
 
 import Foundation
 
-
-fileprivate let userDefaultsKey = "favorite.translations"
-fileprivate typealias FavoritesTranslationsStore = [String: [String: String]]
+typealias FavoritesTranslationsStore = [String: [String: String]]
 
 class TranslationFavoritesService: ObservableObject {
     
+    private let userDefaultsKey = "favorite.translations"
+    
     /**
      A map of favorited translations by language pack and translation id
      [language pack][translationId] = translation id
      */
-    private var favoritedTranslationsByLanguage: FavoritesTranslationsStore = [:]
+    @Published private(set) var favoritedTranslationsByLanguage: FavoritesTranslationsStore = [:] {
+        didSet {
+            UserDefaults.standard.setValue(favoritedTranslationsByLanguage, forKey: userDefaultsKey)
```

Comment:
řádek 67 - to by se mělo odstranit pokud to děláš tady automaticky

Reply:
Ha, díky. :+1: